fe/fe-core/src/main/java/org/apache/doris/catalog/AggregateFunction.java

```diff
@@ -102,6 +102,38 @@ public AggregateFunction(FunctionName fnName, List<Type> argTypes,
                 mergeFnSymbol, getValueFnSymbol, removeFnSymbol, finalizeFnSymbol, false);
     }
 
+    public AggregateFunction(FunctionName fnName, List<Type> argTypes,
```

Comment:
Adding a constructor only for UT is a less elegant behavior. 
If you need to use this variable in UT, but there are some attributes that you can’t use, the better way is new an object and pass in an empty string instead of adding a constructor without a default value.